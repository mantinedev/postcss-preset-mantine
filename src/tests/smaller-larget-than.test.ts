import { testTransform } from './utils';

const smallerThanInput = `
.a {
  @mixin smaller-than 320px {
    background: red;
  }
}
`;

const largerThanInput = `
.a {
  @mixin larger-than 320px {
    background: red;
  }
}
`;

describe('smaller-than', () => {
  it('it transforms smaller-than mixin correctly', async () => {
    const res = await testTransform(smallerThanInput);
    expect(res.css).toMatchSnapshot();
  });
});

describe('larger-than', () => {
  it('it transforms larger-than mixin correctly', async () => {
    const res = await testTransform(largerThanInput);
    expect(res.css).toMatchSnapshot();
  });
});
