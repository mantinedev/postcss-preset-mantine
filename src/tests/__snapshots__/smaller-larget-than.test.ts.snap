// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`larger-than it transforms larger-than mixin correctly 1`] = `
"
@media (min-width: 20em) {
.a {
    background: red
}
}
"
`;

exports[`smaller-than it transforms smaller-than mixin correctly 1`] = `
"
@media (max-width: 19.99375em) {
.a {
    background: red
}
}
"
`;
